src/gui/AutomatableModelView.cpp

```diff
@@ -171,7 +171,12 @@ void AutomatableModelView::unsetModel()
 
 void AutomatableModelView::mousePressEvent( QMouseEvent* event )
 {
-	if( event->button() == Qt::LeftButton && event->modifiers() & Qt::ControlModifier )
+	if( event->button() == Qt::LeftButton &&
+#ifdef LMMS_BUILD_APPLE
+		event->modifiers() & Qt::AltModifier )
+#else
+		event->modifiers() & Qt::ControlModifier )
+#endif
```

Comment:
This code is repeated several times throughout the code. I propose to put it into a helper method with a speaking name. Doing so would also reduce the places where the `#ifdef LMMS_BUILD_APPLE` checks are needed/done.

Reply:
I invite anyone with better understanding of the code to do so and push directly to this branch. ♥️